src/script/interpreter.cpp

```diff
@@ -1699,6 +1727,11 @@ bool SignatureHashSchnorr(uint256& hash_out, ScriptExecutionData& execdata, cons
         ss << tx_to.vin[in_pos].prevout;
         ss << cache.m_spent_outputs[in_pos];
         ss << tx_to.vin[in_pos].nSequence;
+    } else if (input_type == SIGHASH_ANYPREVOUT) {
+        ss << cache.m_spent_outputs[in_pos];
+        ss << tx_to.vin[in_pos].nSequence;
+    } else if (input_type == SIGHASH_ANYPREVOUTANYSCRIPT) {
+        ss << tx_to.vin[in_pos].nSequence;
```

Comment:
yes, `SIGHASH_ANYPREVOUTANYSCRIPT` omits the script and the amount: https://github.com/bitcoin/bips/blob/master/bip-0118.mediawiki#signature-message

Reply:
Perhaps instead of `ANYPREVOUT` and `ANYPREVOUTANYSCRIPT`, it would be better to name them  `ANYPREVOUTLIMITED` and `ANYPREVOUT`? The latter verifies a particular pubkey is signing the output(s); the fomer is more limited, in that it also verifies that the prevout's scriptPubKey and value, and the particular tapscript that's being used.